llvm/include/llvm/CodeGen/GlobalISel/LegalizerHelper.h

```diff
@@ -426,6 +426,7 @@ class LegalizerHelper {
   LegalizeResult lowerMinMax(MachineInstr &MI);
   LegalizeResult lowerFCopySign(MachineInstr &MI);
   LegalizeResult lowerFMinNumMaxNum(MachineInstr &MI);
+  LegalizeResult lowerFMinimum_FMaximum(MachineInstr &MI);
```

Comment:
I'd keep the base expansion separate from the reduction expansions

Reply:
removed from this PR. Will open a separate PR.